src/compiler/compiler.ts

```diff
@@ -45,11 +45,11 @@ export function compile(spec, stats, theme?) {
         name: 'cell',
         type: 'group',
         properties: {
-          enter: {
-            width: layout.cellWidth ?
+          update: {
```

Comment:
For consistency.  (Otherwise, a part of the spec would use enter, another part would use update)